selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java

```diff
@@ -106,7 +118,10 @@ public void checkMoveItem0() throws Exception {
     refactor.clickOnExpandIconTree("/src/main/java");
     refactor.chooseDestinationForItem("p1");
     refactor.clickOkButtonRefactorForm();
+    loader.waitOnClosed();
```

Comment:
You put this these in various places. Do you understand when they are needed?

Reply:
it is needed in places where UI loader can be shown